src/rosbot_system.cpp

```diff
@@ -12,7 +12,7 @@ namespace rosbot_hardware_interfaces
 {
 CallbackReturn RosbotSystem::on_init(const hardware_interface::HardwareInfo& hardware_info)
 {
-  RCLCPP_INFO(rclcpp::get_logger("RosbotSystem"), "Initializing");
+  RCLCPP_INFO(rclcpp::get_logger("rosbot_system"), "Initializing");
```

Comment:
Why not like this?
```suggestion
  RCLCPP_INFO(node_->get_logger(), "Initializing");
```

Reply:
I suggest also change node name from "rosbot_system_node" to simply "rosbot_system"

Reply:
The node_ is created at the end of the `on_init` function. This would cause `segmentation fault`.